scripts/shared/cloud-prepare.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+set -em -o pipefail
+
+source "${SCRIPTS_DIR}/lib/debug_functions"
+source "${SCRIPTS_DIR}/lib/utils"
+
+readonly GATEWAY_LABEL='submariner.io/gateway=true'
+
+### Functions ###
+
+function cloud_prepare() {
+    [[ ${cluster_subm[$cluster]} = "true" ]] || return 0
```

Comment:
If anyone else happens to need a reminder that `=` and `==` are equivalent comparisons in this context:

https://stackoverflow.com/a/20449556/1011749